src/main/resources/templates/db_scripts/schema.json

```diff
@@ -220,6 +230,10 @@
         {
           "fieldName": "customFields",
           "tOps":"ADD"
+        },
+        {
+          "fieldName" : "type",
+          "tOps" : "ADD"
```

Comment:
We don't need fullText and gin index here. 
Just common index, also please make sume testing, create cql query to filter out type='shadow' and make sure that index used.
Firstly you need to check that there is no warning that index missed in mod-users logs.
Secondly you should get generated sql query and invoke EXPLAIN ANALYZE to confirm that index used

Reply:
![image](https://github.com/folio-org/mod-users/assets/90304030/0aa82522-2d5c-4ed5-b2e7-242a30de243d)

Reply:
![image](https://github.com/folio-org/mod-users/assets/90304030/504cfaac-adf8-48a2-a638-899edf39d4ef)